operators/admin/olm-adding-operators-to-cluster.adoc

```diff
@@ -33,4 +33,14 @@ include::modules/olm-installing-specific-version-cli.adoc[leveloffset=+1]
 .Additional resources
 
 * xref:../../operators/admin/olm-upgrading-operators.adoc#olm-approving-pending-upgrade_olm-upgrading-operators[Manually approving a pending Operator upgrade]
+
+include::modules/olm-pod-placement.adoc[leveloffset=+1]
+
+For more specific steps on any of the described procedures, see the following "Additional resources" list:
+
+.Additional resources
+
+* Adding taints and tolerations xref:../../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations-adding_nodes-scheduler-taints-tolerations[manually to nodes] or xref:../../nodes/scheduling/nodes-scheduler-taints-tolerations.html#nodes-scheduler-taints-tolerations-adding-machineset_nodes-scheduler-taints-tolerations[with machine sets]
```

Comment:
Same here about the second xref `.html` -> `.adoc`

Reply:
Oops.